src/v2/spec.yaml

```diff
@@ -663,14 +663,16 @@ components:
               $ref: "#/components/schemas/Channel"
     CastWithInteractionsAndConversations:
       type: object
+      - required:
+            - direct_replies
       allOf:
-        - $ref: "#/components/schemas/CastWithInteractions"
-      properties:
-        direct_replies:
-          type: array
-          items:
-            $ref: "#/components/schemas/CastWithInteractions"
-          description: "note: This is recursive. It contains the direct replies to the cast and their direct replies up to n reply_depth."
+        - $ref: '#/components/schemas/CastWithInteractions'
+          properties:
```

Comment:
Sibling values to ref is ignored in allOf `-` is needed in front of properties since allOf is an array